backend/controller/dal/pubsub.go

```diff
@@ -100,11 +102,34 @@ func (d *DAL) ProgressSubscriptions(ctx context.Context, eventConsumptionDelay t
 		if err != nil {
 			return 0, fmt.Errorf("failed to schedule async task for subscription: %w", dalerrs.TranslatePGError(err))
 		}
+
+		err = d.recordSubscriberCalled(ctx, subscription, subscriber.Sink)
+		if err != nil {
+			logger.Errorf(err, "failed to record subscriber call")
+		}
+
 		successful++
 	}
 	return successful, nil
 }
 
+func (d *DAL) recordSubscriberCalled(ctx context.Context, subscription sql.GetSubscriptionsNeedingUpdateRow, sinkRef schema.RefKey) error {
+	topic, err := d.db.GetTopicByKey(ctx, subscription.Topic)
+	if err != nil {
+		return fmt.Errorf("failed to look up topic by key: %w", dalerrs.TranslatePGError(err))
+	}
+
+	// GetModulesByID can take multiple IDs, so it returns a list of rows. This call
+	// should always return exactly one row, since we pass in one ID.
+	modules, err := d.db.GetModulesByID(ctx, []int64{subscription.ModuleID})
```

Comment:
The module name seems to already be coming in as `subscription.Key.Payload.Module` I think? Also (definitely a nit) wondering if it makes sense to join the topic name above to avoid db roundtrips here?

Reply:
oooo nice catch! fixed